scripts/qa_crowbarsetup.sh

```diff
@@ -2523,6 +2523,11 @@ function enable_ssl_generic()
             fi
             $p "$a['api']['protocol']" "'https'"
         ;;
+        cinder)
+            if iscloudver 7plus; then
```

Comment:
Maybe add a comment about "broken because of bsc#988980" here
so that it easier to find later if this is fixed.